cleanlab/rank.py

```diff
@@ -332,10 +373,20 @@ def get_label_quality_ensemble_scores(
     elif weight_ensemble_members_by == "accuracy":
         weights = np.array(accuracy_list) / sum(accuracy_list)  # Weight by relative accuracy
         if verbose:
-            print("Ensemble members will be weighted by: their relative accuracy")
+            print("Ensemble members will be weighted by their relative accuracy")
             for i, acc in enumerate(accuracy_list):
                 print(f"  Model {i} accuracy : {acc}")
-                print(f"  Model {i} weights  : {weights[i]}")
+                print(f"  Model {i} weight   : {weights[i]}")
+
+        # Aggregate scores with weighted average
+        label_quality_scores = (scores_ensemble * weights).sum(axis=1)
+
+    elif weight_ensemble_members_by == "log_loss_search":
+        weights = neg_log_loss_weights  # Weight by exp(t * -log_loss) where t is found by searching set of T
+        if verbose:
+            print("Ensemble members will be weighted by exp(t * -log_loss)")
```

Comment:
"weighted by exp(t * -log_loss)"
->
"weighted by log-loss between their predicted probabilities and given labels"

(The former output is intimidating for novice user who doesn't know (or need to know) what is happening internally)

Reply:
Good call. Made this change. Thanks.